.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't merge changes in xml files.

Reply:
And package-lock.json

Reply:
should I add these to gitignore?